frontend/__mocks__/mockWorkbench.ts

```diff
@@ -0,0 +1,113 @@
+// /api/k8s/apis/route.openshift.io/v1/namespaces/project/routes/workbench
+
+export const mockWorkbench = {
```

Comment:
mock ... Route?

Reply:
its a mock for a route GET 200 response. It uses [mock service worker](https://mswjs.io/) to apply the mocks to the network calls. I found it makes it easier to control what is going on without having to put mocks on every function that calls the api

Reply:
You called it `mockWorkbench` but it returns a Route object

Reply:
that is whatever was returned by that api route commented above the file.

Reply:
it is no. that is just a name i gave it. I can do something like `mock<resource_type><name>`